python/pyspark/sql/protobuf/functions.py

```diff
@@ -48,8 +48,11 @@ def from_protobuf(
     ----------
     data : :class:`~pyspark.sql.Column` or str
         the binary column.
-    messageName: str
-        the protobuf message name to look for in descriptor file.
+    messageName: str, optional
+        the protobuf message name to look for in descriptor file. Or
+        The Protobuf class name. E.g. ``org.spark.examples.protobuf.ExampleEvent``,
```

Comment:
We need to add a comment about shading requirement. In scala, we are changing this name to `shadedMessageClassName`. See scaladoc update in open PR here: https://github.com/apache/spark/pull/38384/files#diff-93adb6d1143432aff776baed60d056133658635a2d8f9d29d925018fada64ff7R76-R83

Reply:
fixed